code/modules/religion/religion_sects.dm

```diff
@@ -0,0 +1,155 @@
+		if(BP.status == BODYPART_ROBOTIC)
+			to_chat(user, "<span class='warning'>[GLOB.deity] refuses to heal this metallic taint!</span>")
+			return TRUE
+
+	var/heal_amt = 10
+	var/list/hurt_limbs = H.get_damaged_bodyparts(1, 1, null, BODYPART_ORGANIC)
+
+	if(hurt_limbs.len)
+		for(var/X in hurt_limbs)
+			var/obj/item/bodypart/affecting = X
+			if(affecting.heal_damage(heal_amt, heal_amt, null, BODYPART_ORGANIC))
+				H.update_damage_overlays()
+		H.visible_message("<span class='notice'>[user] heals [H] with the power of [GLOB.deity]!</span>")
+		to_chat(H, "<span class='boldnotice'>May the power of [GLOB.deity] compel you to be healed!</span>")
+		playsound(user, "punch", 25, TRUE, -1)
+		SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
+	return TRUE
+
+/datum/religion_sect/puritanism
+	name = "Puritanism (Default)"
+	desc = "Nothing special."
+	convert_opener = "Your run-of-the-mill sect, there are no benefits or boons associated. Praise normalcy!"
+
+/// SECT_TECH
+/datum/religion_sect/technophile
+	name = "Technophile"
+	desc = "A sect oriented around technology."
+	convert_opener = "May you find peace in a metal shell, acolyte.<br>Bibles now recharge cyborgs and heal robotic limbs if targeted, but they do not heal organic limbs. You can now sacrifice cells, with favor depending on their charge."
+	alignment = ALIGNMENT_NEUT
+	desired_items = list(/obj/item/stock_parts/cell)
+	rites_list = list(/datum/religion_rites/synthconversion)
+	altar_icon_state = "convertaltar-blue"
+
+/datum/religion_sect/technophile/sect_bless(mob/living/L, mob/living/user)
+	if(iscyborg(L))
+		var/mob/living/silicon/robot/R = L
+		var/charge_amt = 50
+		if(L.mind?.holy_role == HOLY_ROLE_HIGHPRIEST)
+			charge_amt *= 2
+		R.cell?.charge += charge_amt
+		R.visible_message("<span class='notice'>[user] charges [R] with the power of [GLOB.deity]!</span>")
+		to_chat(R, "<span class='boldnotice'>You are charged by the power of [GLOB.deity]!</span>")
+		SEND_SIGNAL(R, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
+		playsound(user, 'sound/effects/bang.ogg', 25, TRUE, -1)
+		return TRUE
+	if(!ishuman(L))
+		return
+	var/mob/living/carbon/human/H = L
+	var/obj/item/bodypart/BP = H.get_bodypart(user.zone_selected)
+	if(BP.status != BODYPART_ROBOTIC)
+		to_chat(user, "<span class='warning'>[GLOB.deity] scoffs at the idea of healing such fleshy matter!</span>")
+		return TRUE
+	if(BP.heal_damage(5,5,null,BODYPART_ROBOTIC))
+		H.update_damage_overlays()
+	H.visible_message("<span class='notice'>[user] heals [H] with the power of [GLOB.deity]!</span>")
+	to_chat(H, "<span class='boldnotice'>May the power of [GLOB.deity] compel you to be healed!</span>")
+	playsound(user, 'sound/effects/bang.ogg', 25, TRUE, -1)
+	SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
+	return TRUE
+
+/datum/religion_sect/technophile/can_sacrifice(obj/item/I, mob/living/L)
+	if(istype(I,/obj/item/stock_parts/cell))
```

Comment:
Shouldn't this check on `desired_items` instead of `/obj/item/stock_parts/cell`?